### PR TITLE
Javascript javascript1 week3

### PR DESCRIPTION
**Pull request by Aditi-Sawardekar**
Initial Submission

### CODE REVIEW

File: javascript/javascript1/week3/index.html [hunk 1]
```diff
@@ -5,11 +5,18 @@
...
 </head>
 
-<body>
+<body class="container-body">
+    <h1>Smart-ease</h1>
+    <h2>JavaScript1 - Week3 - Homework</h2>
+
+    <section>
+        <h3>Freecode Camp Exercise</h3>
+        <p>Username: Aditi-Sawardekar</p>
```
Comment (Vahablotfi) [inline javascript/javascript1/week3/index.html:20]: Your free code camp is private , I can not see.

File: javascript/javascript1/week3/item-array-removal.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+console.log("Hello World!");
+
```
Comment (Vahablotfi) [inline javascript/javascript1/week3/item-array-removal.js:2]: Everything is nice and clean.

File: javascript/javascript1/week3/when-will-we-be-there.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+console.log("Hello World!");
+
```
Comment (Vahablotfi) [inline javascript/javascript1/week3/when-will-we-be-there.js:2]: Everything is fine.

File: javascript/javascript1/week3/series-duration-of-my-life.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    // Calculate percentage time spent watching a series
+    const percentage = ((100 / averageLifeSpanInMinutes) * totalTimeASeries).toFixed(2)
+    console.log(`${element.title} took ${percentage}% of my life`);
+        
+    //Returned time to the array -> series
+    return totalTimeASeries 
+  })
+
+  // Reduce method on series array to get totalTime
+  const totalTime = series.reduce((currentTotal, element) => {
```
Comment (Vahablotfi) [inline javascript/javascript1/week3/series-duration-of-my-life.js:66]: well done

File: javascript/javascript1/week3/NOnoN0nOYes.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+const allNotes = logOutNotesFormatted(); // should log out the text below
+
+// The note with id: 1, has the following note text: Pick up groceries
+// The note with id: 2, has the following note text: Do laundry
+
+// UNIQUE FEATURE
+
+/* When the activity is done, it will strike through the note*/
+function checkDone (id){
+  for (i=0; i< notes.length; i++){
```
Comment (Vahablotfi) [inline javascript/javascript1/week3/NOnoN0nOYes.js]: Same as here . Also I got the point about `strike()`, but it doesn't work. It shows this `<strike>Do laundry</strike>`

**Comment by Vahablotfi** [comment]
well done, it was good , you have a good understanding, I left few comments, and one more thing I need to add, is that you don't need to use many commits, there were several empty files commits and some commits where you only added Script tag in HTML file. I think, It is not necessary to commit your code after small change , just commit after finishing each step.

File: javascript/javascript1/week3/index.html (hunk 1 above)
Comment (Aditi-Sawardekar) [reply javascript/javascript1/week3/index.html:20]: Hi, My freecode camp profile is public. I have saved it again as public. Please check again

File: javascript/javascript1/week3/item-array-removal.js (hunk 2 above)
Comment (Aditi-Sawardekar) [reply javascript/javascript1/week3/item-array-removal.js:2]: Thank you

File: javascript/javascript1/week3/when-will-we-be-there.js (hunk 3 above)
Comment (Aditi-Sawardekar) [reply javascript/javascript1/week3/when-will-we-be-there.js:2]: Thank you

File: javascript/javascript1/week3/NOnoN0nOYes.js (hunk 5 above)
Comment (Aditi-Sawardekar) [reply javascript/javascript1/week3/NOnoN0nOYes.js]: I will check it...

Comment (Aditi-Sawardekar) [reply javascript/javascript1/week3/NOnoN0nOYes.js]: Hi,

I made some changes in the code to display the note and also the strike.
I read that strike method is Deprecated: This feature is no longer recommended.
Should I use some other method?

**Pull request merged**